scripts/kubestellar

```diff
@@ -256,31 +259,42 @@ function ensure_espw() {
     echo "Finished populating the espw with kubestellar apiexports"   
 }
 
-if [ "$subcommand" == init ]; then
-    ensure_root_compute_configd
-    ensure_espw
-    exit
-fi
-
-# Check mailbox-controller is already running
-if [ $(process_running mailbox-controller) == "running" ]
-then
-    echo "An older deployment of mailbox-controller is already running - terminating it ...."
-    pkill -f mailbox-controller
-fi
-
-# Check kubestellar-where-resolver is already running
-if [ $(process_running "kubestellar-where-resolver") == "running" ]
-then
-    echo "An older deployment of kubestellar-where-resolver is already running - terminating it ...."
-    pkill -f "kubestellar-where-resolver"
-fi
+# current ws at start does not matter, is imw1 on return
+function ensure_imw1() {
+    kubectl ws root &> /dev/null
+    if kubectl get ws imw1 &> /dev/null
+    then kubectl ws imw1 &> /dev/null
+    else kubectl ws create imw1 --enter &> /dev/null
+    fi
+    
+    if ! kubectl get apibinding "edge.kubestellar.io" &> /dev/null; then
+	kubectl kcp bind apiexport root:espw:edge.kubestellar.io
+    fi
```

Comment:
Do you want to go back to root after you finishe

Reply:
You are debating the behavior of the internal script function `ensure_imw1`, right? #1057, which differs a little in this script, has been merged instead of this PR. If you have a better idea for the internal details of the merged script, please open an issue or PR.